libpod/runtime_img.go

```diff
@@ -645,36 +651,36 @@ func (r *Runtime) PullImage(imgName string, options CopyOptions) error {
 		// could be trying to pull from registry with short name
 		pullStructs, err = getRegistriesToTry(imgName, r.store, r.config.ImageDefaultTransport)
 		if err != nil {
-			return errors.Wrap(err, "error getting default registries to try")
+			return "", errors.Wrap(err, "error getting default registries to try")
 		}
 	} else {
 		pullStructs, err = r.getPullListFromRef(srcRef, imgName, sc)
 		if err != nil {
-			return errors.Wrapf(err, "error getting pullStruct info to pull image %q", imgName)
+			return "", errors.Wrapf(err, "error getting pullStruct info to pull image %q", imgName)
 		}
 	}
-
 	policyContext, err := getPolicyContext(sc)
 	if err != nil {
-		return err
+		return "", err
 	}
 	defer policyContext.Destroy()
 
 	copyOptions := common.GetCopyOptions(options.Writer, signaturePolicyPath, &options.DockerRegistryOptions, nil, options.SigningOptions, options.AuthFile)
 
 	for _, imageInfo := range pullStructs {
-		if options.Writer != nil {
+		// Print the following statement only when pulling from a docker or atomic registry
```

Comment:
Thanks for the changes here, much clearer (or perhaps I've more tea in my system now!  :^)  )